portal/templates/layouts/errors/400.html

```diff
@@ -0,0 +1,15 @@
+{% extends 'layouts/errors/error_page.html' %}
+
+{% block title %}Bad Request 400{% endblock %}
+
+{% block content %}
+<div class="error_pages">
+
+<h3>The request you attempted is Bad and not allowed</h3>
+
+<h3>Either press the back button on the browser or click the home link below</h3>
```

Comment:
Remember, an h3 is a section heading, so semantically these shouldn't both be headings with nothing in between.  Either they should both be p tags, or maybe the first could act as a heading the second be a p.
This goes for all the error templates